bots/zephyr_mirror_backend.py

```diff
@@ -204,8 +212,7 @@ def update_subscriptions():
     classes_to_subscribe = set()
     for stream in public_streams:
         zephyr_class = stream.encode("utf-8")
-        if (options.shard is not None and
-                not hashlib.sha1(zephyr_class).hexdigest().startswith(options.shard)):
+        if (options.shard is not None and not hashlib.sha1(zephyr_class).hexdigest().startswith(options.shard)):
```

Comment:
The Travis error (the E129 rule from PEP8) occurred on an [older commit](6f0423a3bb1692a6ea678fb831bfab1fd1946e83) where the second line wasn't indented so far, and if @anirudhjain75 now puts these the way they are after a6f91064a2b6aab50153afd80a5f4833c3a8ea85 the PEP8 tests should pass.